assets/README.md

```diff
@@ -1,12 +1,16 @@
-# e
-- `assets`          assets root dir
-    - `_img`        source images dir
-    - `img`         destination images dir
+# Assets
```

Comment:
I will see about it. But, specially for the images, files can't be modified in place. If not  `img` and `_img` it will have to be `img/src` and `img/dest`. The good thing of  `_img` is that it's completely optional for the user.

Reply:
Yup I understand. I mean bottom line to avoid gulp re-compressing of image, commit only the newly compressed ones it would be the best deal to me.

Reply:
gulp-imagemin does that in a bizarre away. It will not reoptimize an already optimized image but still it will make modifications to the file. Probably just metadata but mutable metadata so those files have to be staged and commited every time.

Reply:
Indeed 🤔 I'll read their documentation